.github/workflows/main.yml

```diff
@@ -141,7 +141,7 @@ jobs:
         architecture: 'x64'
     - name: Install Python packages
       run: |
-        python -m pip install wheel setuptools cpplint pylint
+        python -m pip install wheel setuptools cmakelint cpplint pylint
```

Comment:
I know that https://github.com/dmlc/xgboost/issues/9630#issuecomment-1751915022 recommended using conda/mamba, but since this job was already not using those tools to manage its environment, I chose to just add `cmakelint` here. Let me know if you'd like me to change that.